api/pkg/service/auth/auth_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/h2non/gock.v1"
+
+	"github.com/tektoncd/hub/api/gen/auth"
+	"github.com/tektoncd/hub/api/pkg/testutils"
+)
+
+func TestLogin(t *testing.T) {
+	tc := testutils.Setup(t)
+	testutils.LoadFixtures(t, tc.FixturePath())
+
+	defer gock.Off()
+
+	gock.New("https://github.com").
+		Post("/login/oauth/access_token").
+		Reply(200).
+		JSON(map[string]string{
+			"access_token": "test-token",
+		})
+
+	gock.New("https://api.github.com").
+		Get("/user").
+		Reply(200).
+		JSON(map[string]string{
+			"login": "test",
+			"name":  "test-user",
+		})
+
+	var token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9." +
```

Comment:
do you think we can use a package level `const ValidToken string = ".... "` for this?